e2e/motoko/Main.mo

```diff
@@ -1,202 +1,214 @@
 import EvmRpcCanister "canister:evm_rpc";
+import EvmRpcFidicuaryCanister "canister:evm_rpc_fiduciary";
 
-import Principal "mo:base/Principal";
-import Evm "mo:evm";
+import Blob "mo:base/Blob";
 import Debug "mo:base/Debug";
+import Cycles "mo:base/ExperimentalCycles";
+import Principal "mo:base/Principal";
 import Text "mo:base/Text";
-import Blob "mo:base/Blob";
-import Cycles "mo:base/ExperimentalCycles"
+import Evm "mo:evm";
 
 shared ({ caller = installer }) actor class Main() {
-    let mainnet = Evm.Rpc(
-        #Canister EvmRpcCanister,
-        #Service {
-            hostname = "cloudflare-eth.com";
-            network = ? #EthMainnet;
-        },
-    );
-
     public shared ({ caller }) func test() : async () {
         assert caller == installer;
 
-        let source = #Service {
-            hostname = "cloudflare-eth.com";
-            chainId = ?(1 : Nat64); // Ethereum mainnet
-        };
-        let json = "{\"jsonrpc\":\"2.0\",\"method\":\"eth_gasPrice\",\"params\":null,\"id\":1}";
-        let maxResponseBytes : Nat64 = 1000;
-
-        // `requestCost()`
-        let cyclesResult = await EvmRpcCanister.requestCost(source, json, maxResponseBytes);
-        let cycles = switch cyclesResult {
-            case (#Ok cycles) { cycles };
-            case (#Err err) {
-                Debug.trap("unexpected error for `request_cost`: " # (debug_show err));
+        let canisterDetails = [
+            (EvmRpcCanister, "default", 13, 521_498_000),
+            (EvmRpcFidicuaryCanister, "fiduciary", 28, 1_123_226_461),
```

Comment:
It is not clear what these numbers mean (readers may be able to guess what 13 and 28 mean but not the other numbers).
How about either adding a comment or replacing the plain numbers with meaningfully named constants?

Reply:
Added a comment to clarify. There's also a line directly after this which shows the variable names for each tuple entry, although I see how it could be easy to miss.